src/main/java/com/epam/training/toto/service/TotoService.java

```diff
@@ -1,4 +1,142 @@
 package com.epam.training.toto.service;
 
+import com.epam.training.toto.domain.Outcome;
+import com.epam.training.toto.domain.Round;
+import com.epam.training.toto.service.parser.CSVParser;
+import com.epam.training.toto.service.parser.util.OutcomeParser;
+
+import java.io.IOException;
+import java.text.NumberFormat;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.*;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
 public class TotoService {
+    public static final Integer NUMBER_OF_GAMES = 14;
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy.MM.dd.");
+    private final NumberFormat numberFormat = NumberFormat.getCurrencyInstance(new Locale("uk", "UA"));
+
+    private Map<LocalDate, Round> data;
+
+    public TotoService() {
+        try {
+            data = new CSVParser().readData();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private Integer getTheLargestPrize() {
+        List<Integer> allPrizes = data.values()
+                .stream()
+                .map(round -> round.getHits())
+                .flatMap(Collection::stream)
+                .map(hit -> hit.getPrizeForTheGames())
+                .collect(Collectors.toList());
+
+        return Collections.max(allPrizes);
+    }
+
+    private List<Map<Outcome, Long>> getDistributionOfResultsForEachRound() {
+        List<Map<Outcome, Long>> distributionForEachRound = new ArrayList<>(2 * data.size());
+        data.values()
+                .forEach(round -> distributionForEachRound.add(round.getOutcomes()
+                        .stream()
+                        .collect(Collectors.groupingBy(Function.identity(), Collectors.counting())))
+                );
+        return distributionForEachRound;
+    }
+
+    public void printTheLargestPrize() {
+        System.out.println("======== The largest prize: " +
```

Comment:
I'd suggest to use wrapper for all console output. Also regular java logger can be used instead.

Reply:
done